{{cookiecutter.project_name}}/requirements-dev.in

```diff
@@ -0,0 +1,31 @@
+#
+# Development requirements
+#
+-r requirements.in
+
+# Dev tools
+ipython
+ipdb
+pip-tools
+django-debug-toolbar
+
+# Testing
+django-webtest
+doc8
+flake8
+model-bakery
+pytest
+pytest-black
+pytest-cov
+pytest-django
+pytest-isort
+pytest-flake8
+pytest-mypy
+
+# Documentation
+sphinx
+sphinx_rtd_theme
+
+# Deployment
+fabric
```

Comment:
We should probably remove fabric in favour of invoke.

Reply:
I believe invoke isn't a straight replacement for fabric, but is used alongside fabric 2? The `upgrade` branch has fabric 2, probably best to address this there.

Reply:
I was wondering, with this and the testing deployments bit in upgrade, with us splitting out k8s etc. Should build and deployment also be split to another project repo or is there still value having it in here?

Reply:
Lets make it js- and deployment-agnostic for now - strip out anything about building js, fabric, deployment; we can add them back in when we need them.

Reply:
Removed invoke and fabric.